src/components/RequestCodePanel.astro

```diff
@@ -0,0 +1,23 @@
+---
+import { HTTPSnippet } from 'httpsnippet';
+const { title = 'Request', filename } = Astro.props;
+import { Code } from 'astro:components';
+import CodePanel from './CodePanel.astro';
+
+const file = await import(`../content/api/_code-snippets/dist/${filename}.json`);
```

Comment:
@MeganSteenkamp Suggestion: Store generated HAR files outside of the `src` directory. Keeps things cleaner for local development

Reply:
@tom-blake I've moved this to a top level directory called `code-snippets` - let me know if you have a suggestion on where you were wanting the generated files to go 🎉